vmdb/app/helpers/container_service_helper/textual_summary.rb

```diff
@@ -83,4 +84,14 @@ def textual_container_groups
     end
     h
   end
+
+  def textual_container_routes
```

Comment:
For this to work you also have to add `container_services` in `vmdb/app/views/container_service/show.html.haml`, something like:

    if %w(container_groups container_services container_routes).include?(@display)